services/common-controls.yaml

```diff
@@ -70,16 +70,18 @@ controls:
     test_requirements:
       - id: CCC.C02.TR01
         text: |
-          The service encrypts all stored data at rest using industry-standard encryption algorithms (e.g., AES-256).
+          The service encrypts all stored data at rest using
+          industry-standard encryption algorithms (e.g., AES-256).
         tlp_levels:
           - tlp_clear
           - tlp_green
           - tlp_amber
           - tlp_red
       - id: CCC.C02.TR02
         text: |
-          Admin users can verify and audit encryption status for stored data at rest,
-          including verification of key management processes.
+          Admin users can verify and audit encryption status for
+          stored data at rest, including verification of key
+          management processes.
```

Comment:
Do we want to make the same change here to remove admin users from description?

Reply:
Done now!